src/w2utils.js

```diff
@@ -2219,6 +2222,19 @@ class Utils {
             })
         })
     }
+
+    debounce(func, wait) {
```

Comment:
I like the idea of `debounce` here, it makes sense to have it in w2utils. Can we use this implementation of `debounce`, https://www.freecodecamp.org/news/javascript-debounce-example/.

I checked it keeps the right `this`, it has arrow functions and it is clean